lib/src/wallet/transactions.rs

```diff
@@ -523,7 +528,35 @@ impl WalletTxns {
                         });
                 }
             }
-            WalletNullifier::Orchard(_) => todo!(),
+            WalletNullifier::Orchard(nullifier) => {
+                if wtx
+                    .spent_orchard_nullifiers
+                    .iter()
+                    .find(|nf| **nf == nullifier)
+                    .is_none()
+                {
+                    wtx.add_spent_nullifier(WalletNullifier::Orchard(nullifier), value)
+                }
+
+                // Since this Txid has spent some funds, output notes in this Tx that are sent to us are actually change.
+                self.check_notes_mark_change(&txid);
+
+                // Mark the source note's nullifier as spent
+                if !unconfirmed {
```

Comment:
This definition of `confirmed` means that the `nullifier` is in a block?  And therefore can be marked as `spent`?